junit5/src/main/java/sunstone/api/Sunstone.java

```diff
@@ -0,0 +1,20 @@
+package sunstone.api;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import sunstone.core.SunstoneExtension;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Extension that enable Sunstone to deploy, inject and manage Cloud resources.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE})
+@ExtendWith({SunstoneExtension.class})
```

Comment:
Is it possible to ditch this annotation and use `@ExtendWith({SunstoneExtension.class})` on all the other annotations you've introduced? The point is to save typing one more annotation for the user.

Reply:
Good point. I will get rid of `Sunstone` annotation.